com.unity.shadergraph/Editor/GraphUI/GraphViewFactoryExtensions.cs

```diff
@@ -2,29 +2,34 @@
 using UnityEditor.GraphToolsFoundation.Overdrive.BasicModel;
 using UnityEditor.ShaderGraph.GraphUI.DataModel;
 using UnityEditor.ShaderGraph.GraphUI.GraphElements;
+using UnityEditor.ShaderGraph.GraphUI.GraphElements.Views;
 
-namespace UnityEditor.ShaderGraph.GraphUI.Factory
+namespace UnityEditor.ShaderGraph.GraphUI
 {
     [GraphElementsExtensionMethodsCache(typeof(GraphView))]
-    public static class ShaderGraphViewFactoryExtensions
+    public static class GraphViewFactoryExtensions
     {
-        public static IModelUI CreateGraphDataNode(this ElementBuilder elementBuilder, CommandDispatcher store,
+        public static IModelUI CreateNode(
+            this ElementBuilder elementBuilder,
             GraphDataNodeModel model)
         {
             var ui = new GraphDataNode();
             ui.SetupBuildAndUpdate(model, elementBuilder.View, elementBuilder.Context);
             return ui;
         }
 
-        public static IModelUI CreateRedirectNode(this ElementBuilder elementBuilder, CommandDispatcher store,
+        public static IModelUI CreateRedirectNode(
```

Comment:
I have a feeling that we'll need to change the `CreateRedirectNode()` call to `CreateNode` as well for it to work, but we don't need redirect nodes right now anyway so this can wait

Reply:
I'll take a look at what needs to change.

Reply:
I see what you are saying. The `setupBuildAndUpdate` needs to be updated for `RedirectNode`. 

That is outside of the scope for this PR, so I added a jira task for it https://jira.unity3d.com/browse/GSG-892